media/stump-radio/README

```diff
@@ -1,6 +1,8 @@
 a simple mplayer-based radio for the window manager StumpWM
```

Comment:
I suggest using org-mode instead on plaintext. Most packages in this repo use it.

Reply:
Sorry, my friend, this is me spare time programming. I hate these
half-baked tag languages like org-mode or Markdown when a simple text
file is sufficient and more beautiful in its simplicity. I switched
to StumpWM to use less of this clutter. Actually, by authoring the file
I was surprised how much I enjoyed writing just a simple text README
file again. I beg your pardon but I would prefer not to go back.

All these modules are self-sufficient and thus the stumpwm-contrib
repositry a collection of loose things compared, for example, to
the stumpwm repository. So consider these two new modules a gift and
take them or leave them. I am happy if they stay just in my fork
(or just in my private use for that matter), and I am not offended
if you just decline this pull request.

(Your other comment I have addressed although it is marked outdated
now because of the commits.)

Reply:
Thats fine then. There are a few packages that have a `README.txt` for plaintext, maybe just rename to match.

Reply:
Keep it as is, those README.txt files are artifacts of make-package, not any standard.

Reply:
Good